pyproject.toml

```diff
@@ -23,7 +23,8 @@ python = "^3.8"
 oauthlib = "~=3.1.0"
 requests-oauthlib = "~=1.3.0"
 aiofiles = ">=0.7,<0.9"
-zeebe-grpc = "^8.0.4.post1"
+zeebe-grpc = "8.4.0"
```

Comment:
This pins the version to `8.4.0`, we probably want to be more flexible with `^8.4.0` to support versions like `8.4.1` or `8.5.3`

Reply:
Fixed.